### ISSUE TITLE
About evaluation metric

### ISSUE THREAD

**Issue by minjaelee-postech** [aim-uofa/Matcher#31]
In matcher > common > logger.py 's class AverageMeter

It seems that there is a slight issue with the AverageMeter function. During initialization, a tensor of size [2, self.nclass] is initialized using torch.zeros, which causes the performance to be divided by twice the number of the entire dataset. As a result, when I modified your code and conducted inference, the recorded performance was approximately doubled.
